src/platform/utilities/feature-toggles/flipper-client.js

```diff
@@ -0,0 +1,90 @@
+// import environment from 'platform/utilities/environment';
+const TOGGLE_VALUES_PATH = '/toggle.json';
+const TOGGLE_POLLING_INTERVAL = 5000;
+
+let flipperClientInstance;
+
+function FlipperClient({
+  // host = environment.API_URL,
```

Comment:
TBD- waiting on some `vets-api` stuff. I'll probably change this in another PR. for now if you want to test this you can toss `/toggles.json` in your `localhost` build with this: 

```
{
    "meta": {
        "updated": "2012-04-23T18:25:43.511Z"
    },
    "toggleValues": {
        "facilityLocatorShowCommunityCares": true
    }
}
```

You can change the false/ true value and refresh to change if this appears. Ideally this would be coupled with API changes, as well (currently CC is off for the default search [all facilities] on the service because it's too slow.)

<img width="346" alt="Screen Shot 2019-08-05 at 19 01 42" src="https://user-images.githubusercontent.com/4998130/62503931-9b0f2200-b7b3-11e9-9f52-45df3a9ba98c.png">